tfx/orchestration/portable/beam_executor_operator.py

```diff
@@ -31,6 +33,33 @@
   _BeamPipeline = Any
 
 
+def _resolve_beam_args_from_env(beam_pipeline_args, beam_pipeline_args_from_env) -> list:
+  resolved_beam_pipeline_args_from_env = []
+
+  for beam_pipeline_arg_from_env, env_var in beam_pipeline_args_from_env.items():
+    # If an arg is already present in beam_pipeline_args, it should take precedence
+    # over env vars.
+    if any("--{}=".format(beam_pipeline_arg_from_env) in beam_pipeline_arg for beam_pipeline_arg in beam_pipeline_args):
```

Comment:
Nit pick: f'--{beam_pipeline_arg_from_env}=' is preferred. (We need to use quotes consistently, too.) And beam_pipeline_arg.startswith(...) might be more strict.